### ISSUE TITLE
Get url for video content

### ISSUE THREAD

**Issue by lnxpck** [GoGoCarl/paperclip-cloudinary#2]
Can you tell me how change the resource_type (for video by example).

I tried this one but it doesn't work : 
o.animation.url :original, {:resource_type => :video}

"http://res.cloudinary.com/vdcl6vbxx/image/upload/overlay_90aa2ab0-1ae8-0134-7d7c-0242ac110001.webm"

BTW, i think somethings is wrong with the "url" method :

```
      def url style_or_options = default_style, options = {}
        if style_or_options.is_a?(Hash)
          options.merge! style_or_options
          style = default_style
        else
          style = style_or_options
        end
        ::Cloudinary::Utils.cloudinary_url path(style)
      end
```

options is never used in Cloudinary::Utils.cloudinary_url ?!
and i'm not sure to undestrand the aim of "style" parameter (and "default_style").

Thx

**Comment by GoGoCarl** [comment]
Hi @plesaint.  I can certainly have a look at the resource_type parameter.  I don't think I accounted for that particularly, so if it's something a bit out of band, I may need to add something to support that.  I'll look into it.

Regarding the style parameter, this is moreso for images, when you would want to have different "styles" for the same image.  For example, an image in a photo gallery may have two styles, one being a thumbnail to display in a gallery, and another for an enlarged version.  I could imagine, perhaps, a video having styles too -- say a low-quality version, an HD version, and perhaps even a "snippet" version.

Styles are optional, which is why the first parameter is "style_or_options", as the url call could be made as `url(opts)` or `url(:thumbnail, opts)`, or simply `url` (without options).

And yes, the `cloudinary_url` method has no way of receiving path options from paperclip-cloudinary.  Certainly seems like a useful thing to be able to provide, so I'll add support for it.  I suppose doing this in a general way would also mean you would be able to supply the `:resource_type` parameter.

**Comment by GoGoCarl** [comment]
@plesaint, an update on this.  Working on an update that should accommodate this.  Should be merged this week.  If you'd like to get an early jump and see how it works for your project, or provide any feedback prior to merging, you are more than welcome to do so:

https://github.com/GoGoCarl/paperclip-cloudinary/tree/v1.2.0

Details are in the README there, but the update will allow you to specify these URL options in your `has_attached_file` configuration as well as in-line.  For your example, you'd have:

``` ruby
o.animation.url :original, { :cloudinary => {:resource_type => :video} }
```

I'll update again after this is tested, merged, and deployed.

Thanks for the great suggestion!

**Comment by GoGoCarl** [comment]
@plesaint Hi Patrick, this has been merged and released under version 1.2.0.  Thanks again for the great suggestion.  Please let me know if you have any trouble with this new feature.
